lib/rubies/game.rb

```diff
@@ -71,9 +83,14 @@ def questioner(current)
     end
 
     def eprinter(error)
+      encoding_options = {
+      :invalid           => :replace,  # Replace invalid byte sequences
+      :undef             => :replace,  # Replace anything not defined in ASCII
+      :replace           => '',        # Use a blank for those replacements
+      }
       puts
       puts "Sorry, that code resulted in an error:".colorize(:light_red)
-      puts "#{error}".colorize(:red)
+      puts "#{error}".encode(Encoding.find('ASCII'), encoding_options).colorize(:red)
```

Comment:
Line is too long. [85/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not write to stdout. Use Rails' logger if you want to log.